### ISSUE TITLE
Can't change location from inside the app

### ISSUE THREAD

**Issue by joshboldman** [istornz/iPokeGo#270]
Not entirely sure when it stopped working, but I can't seem to change the scan location from inside the app. I'm using PokemonGo-Map, and I can still change the location if I log in directly to the server through Safari.

If I long press in the app, I get the haptic feedback, and the dot moves, but the scan just continues at the original location. I know there were some changes to how user location was handled recently, so I'm not sure if it's a bug or if I'm just doing something wrong.

**Comment by istornz** [comment]
What is your iPokeGO version and PokemonGO-Map version ?

**Comment by joshboldman** [comment]
Ok, so I was just checking into the versions and everything, and then I realized the problem. I'm using about 20 google accounts to login via the command line, and the first one in my list had been banned. My guess is that since it had been banned, it couldn't handle the new location that iPokeGo was trying to send it. So I removed that account from the command line, and everything works perfect.

But to answer your question, I'm running the most recent git for both iPokeGo 2.2 and PokemonGO-Map as of about 10 minutes ago.
